client/app/styles/main.styl

```diff
@@ -732,20 +732,20 @@ body
         box-sizing: border-box
         width: 180px
         height: 180px
-        padding: 20px 20px 10px 20px
+        padding: 25px 20px 10px 20px
         margin-right: 10px
 
         img.svg
-            border-radius: 200px
-            padding: 10px
-            width:140px
-            height:140px
+            border-radius 200px
+            padding 15px
+            width 130px
+            height 130px
 
```

Comment:
Half of the code uses ":" and the other half doesn't. I have been removing them whenever I can because it doesn't add anything at all. It's been kind of a crusade for me. Let's bring that up into the cozy-guildelines "debate"!